ios/brave-ios/README.md

```diff
@@ -1,113 +1,22 @@
-![Build](https://github.com/brave/brave-ios/workflows/Build/badge.svg?branch=development)
+This folder contains the contents of what used to be in the brave-ios repo.
 
-Brave for iOS 🦁
-===============
+## Building the Brave iOS app:
 
-Download on the [App Store](https://apps.apple.com/app/brave-web-browser/id1052879175).
+Unlike brave-core, the code inside this folder does not build using GN and `npm run build` commands.
 
-This branch (development)
------------
+1. Ensure you have brave-core fully set up already including running an `init` with the `--target_os=ios` argument supplied.
+2. Run the bootstrap shell script found in `ios/brave-ios/scripts`
+3. Open the `Client.xcodeproj` file in Xcode found in `ios/brave-ios/App`
```

Comment:
I think these two steps should be wrapped in some `npm run ios_open_project` command (feel free to change the name) that will bootstrap and open `xcodeproj` automatically.

everything from `ios/brave-ios/scripts/bootstrap.sh` can be implemented in the npm command. This also means  `npm run ios_pack_js` can be replaced with a direct `webpack` module call from the script.